andela/ah-the-answer-backend
# #164829374 Add articles pagination

__What does this PR do?__
Adds pagination to the articles.

__Description of the tasks to be completed__

Add functionality that returns at most ten articles per page.


__How should this be manually tested?__

- Clone the repo, and navigate to the root directory.
- Fetch all the branches and checkout into this feature's branch using the commands below
         -`git fetch --all`
         -`git checkout ft-pagination-articles-#164829374`

- Create a virtual environment and then activate the environment  as shown below
        - `virtualenv venv`
        - `source venv/bin/activate`

- Install all the required dependencies using pip by running
        - `pip install -r requirements.txt`
- Run the Django server to serve up the app by typing
        - `python manage.py runserver`

- Navigate to the endpoint shown below on postman
- Switch the pages by copy-pasting the next or the previous URL on the output data on postman
- The output should be a maximum of ten articles

<img width="1218" alt="Screenshot 2019-04-18 at 07 18 15" src="https://user-images.githubusercontent.com/22986772/56336783-38652c80-61aa-11e9-8db9-0599576d2088.png">
